mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -182,6 +182,11 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 								},
 							},
 						},
+						"time_zone": {
```

Comment:
It looks like this field can't be updated, which means that it's currently causing the new test to fail with `Error 400: Invalid request: Time zone cannot be changed after an instance is created`.

To fix this, you need to add `ForceNew` to this field schema so that changing it will trigger instance recreation.

Reply:
Added